source/material_model/grain_size.cc

```diff
@@ -1211,6 +1334,35 @@ namespace aspect
                              Patterns::Bool (),
                              "This parameter determines whether to use bilinear interpolation "
                              "to compute material properties (slower but more accurate).");
+          prm.enter_subsection("Grain damage partitioning");
+          {
+            prm.declare_entry ("Minimum partitioning temperature", "1600",
+                               Patterns::Double (0.),
+                               "This parameter determines the temperature at which the computed coefficient of shear energy "
+                               "partitioned into grain damage is minimum. This is used in the pinned state limit of the grain "
+                               "size evolution. One choice of this parameter is the mantle temperature at the ridge axis, "
+                               "see Mulyukova and Bercovici (2018) for details.");
+            prm.declare_entry ("Maximum partitioning temperature", "283",
```

Comment:
After seeing your Assert below, I now realize these names are not intuitive (i.e., a user would expect the minimum temperature to be smaller than the maximum temperature). 
Do you think it makes sense to come up with better names?